pom.xml

```diff
@@ -11,7 +11,7 @@
 
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>pipeline-graph-analysis</artifactId>
-    <version>1.3-SNAPSHOT</version>
+    <version>1.3-optimized-SNAPSHOT</version>
```

Comment:
Custom version to ensure we don't accidentally pull in the wrong dependency when benchmarking (also true below).  Will be reverted before merge.